Town-Of-Salem/src/main/java/com/github/raresp/proiectip/TownOfSalem/models/GameState.java

```diff
@@ -0,0 +1,8 @@
+package com.github.raresp.proiectip.TownOfSalem.models;
+
+public enum GameState {
+    Discussion,
```

Comment:
tbh eu le-as face pe astea uppercase ca sa se vada clar ca sunt constante, but it works as it is